components/table/interface.tsx

```diff
@@ -99,7 +104,7 @@ export interface TableProps<T> {
   expandRowByClick?: boolean;
   onExpandedRowsChange?: (expandedRowKeys: string[] | number[]) => void;
   onExpand?: (expanded: boolean, record: T) => void;
-  onChange?: (pagination: TablePaginationConfig | boolean, filters: string[], sorter: Object) => any;
+  onChange?: (pagination: TablePaginationConfig | boolean, filters: Record<keyof T, string[]>, sorter: Record<keyof T, SorterResult>) => any;
```

Comment:
It's just `sorter: SorterResult`, right?

Reply:
No, when I use more than one sorter, it is `Record<keyof T, SorterResult>`. Is it just `SorterResult` in one sorter case?

Reply:
Only one sorter can apply at the same time.